src/monitoring_service/states.py

```diff
@@ -1,11 +1,13 @@
 from dataclasses import dataclass, field
 from typing import Iterable, Optional
 
+from eth_typing.evm import HexAddress
```

Comment:
Mh, a bit ugly to import this as well. Is this what the contracts use?

Reply:
Yes, since they don't have access to Raiden's types.